sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -104,9 +110,9 @@ def __init__(
       self,
       model_handler: ModelHandler[ExampleT, PredictionT, Any],
       clock=time,
-      **kwargs):
+      extra_runinference_args: Optional[Dict[str, Any]] = None):
```

Comment:
What about `extra_kwargs`? These are still kwargs, and I don't think `runinference` adds anything.

Could you also document this in the docstring?

Reply:
Changed, and added.